acc/kernels/constant_qe.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from numba import cuda, float64
+from numba.cuda.random import xoroshiro128p_uniform_float32
+
+from .. import vec3
+from ..config import get_numba_floattype
+
+NB_FLOAT = get_numba_floattype()
+
+epsilon = 1e-1
+
+neutron_mass = 1.6749286e-27
```

Comment:
These look good. But could you replace them with code reusing what is implemented in [mcni.utils.conversion](https://github.com/mcvine/mcvine/blob/master/packages/mcni/python/mcni/utils/conversion.py)?